config/config.yml.template

```diff
@@ -31,6 +35,13 @@ collections:
     name_mapping: 28 Days-Weeks Later
   Anaconda:
     tmdb_id: 105995, 336560
+  Most Popular Movies (30 Days):
+    sync_mode: sync
+    collection_mode: show_items
+    tautulli:
+      list_type: popular
```

Comment:
I'm fine with a `list_type` attribute instead of `tautulli_popular`/`tautulli_watched`